src/Token.php

```diff
@@ -37,12 +35,7 @@ final class Token
     const INTERPOLATION_START_TYPE = 10;
     const INTERPOLATION_END_TYPE = 11;
 
-    /**
-     * @param int    $type   The type of the token
-     * @param string $value  The token value
```

Comment:
the type of `$value` should still be documented

Reply:
I don't think so as this is mixed.